packages/web3-shared/src/hooks/useResolution.ts

```diff
@@ -0,0 +1,13 @@
+import { Resolution } from '@unstoppabledomains/resolution'
+import { useAsyncRetry } from 'react-use'
+import { useChainId } from '.'
+
+export function useResolution(ens: string) {
+    const chainId = useChainId()
+    const resolution = new Resolution()
```

Comment:
`Resolution` constructor accepts a configure object which specify network. You might need to test on other networks.